erpnext/selling/doctype/customer/customer.py

```diff
@@ -100,7 +100,8 @@ def validate(self):
 	@frappe.whitelist()
 	def get_customer_group_details(self):
 		doc = frappe.get_doc("Customer Group", self.customer_group)
-		self.accounts = self.credit_limits = []
```

Comment:
this only worked because `doc.append` always re-initialised the table if the value was falsy 🤣